modules/apps/portal-search-opensearch2/portal-search-opensearch2-impl/src/test/java/com/liferay/portal/search/opensearch2/internal/OpenSearchIndexWriterTest.java

```diff
@@ -106,6 +106,21 @@ public void testPartiallyUpdateDocumentDoesNotRemoveFields()
 		_assertOnlyOne(Field.TITLE, "text");
 	}
 
+	@Test
+	public void testPartiallyUpdateUpsertsMissingDocument()
```

Comment:
Maybe rename `testPartiallyUpdateDocumentUpsertsMissingDocument` as we are testing the `partiallyUpdateDocument` method right?